components/src/core/score-card/score-card.component.scss

```
@@ -3,7 +3,7 @@
 }
 
 // Styles applied to supplied action items.pxb-score-card-
-:host ::ng-deep .pxb-score-card-action-items * {
+.pxb-score-card-action-items * {
```

We might want to change this to be a little more restrictive in what it applies to (just direct children):
```suggestion
.pxb-score-card-action-items > * {
```